src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,20 @@
+.watch {
+  height: $watch-size;
+  width: $watch-size;
+  border: 5px dotted black;
+  border-radius: 50%;
+  align-self: center;
+  display: flex;
+  justify-content: center;
+
+  &__center {
+    align-self: center;
+    position: absolute;
+  }
+
+  &__minute-axis {
+    align-self: center;
+    position: absolute;
+    justify-content: center;
```

Comment:
think this property `justify-content: center;` is redundant here if you using an `position: absolute`